data/the_women.yml

```diff
@@ -140,6 +140,18 @@
   youtube1: "l4KCEoa29GE"
   youtube1_title: "Crafting the Perfect Computed Property"
 
+- name: "Jen Weber"
+  job_title: "UI/UX Developer at BioBright"
+  location: "Boston, MA"
+  bio: "Jen is a UI/UX Developer at BioBright, a startup that builds tools to advance scientific research. She learned to code in 2016 after changing careers. She is fiercely devoted to encouraging, engaging, and promoting new developers - especially those from non-traditional backgrounds. She is working hard to become a technical contributor to open source projects."
```

Comment:
Pfft. She's _already_ a technical contributor! I say there isn't really a way to contribute to something as technical as a JS framework _without_ it counting as technical, regardless of the exact contents of your contributions.

Might I be so bold as to suggest something like this as an alternative: "She's enjoying contributing to open source projects, including of course Ember."? TL;DR I definitely saw a PR or three come through from you, plus merges: own it 😁 

You don't need to specifically mention Ember, I was just hustling to find a second half of the sentence and it's bedtime ❤️